packages/react-examples/src/react-charting/TreeChart/TreeChart.ThreeLayerCompact.Example.tsx

```diff
@@ -98,14 +94,29 @@ export class TreeChartThreeLayerCompactExample extends React.Component<{}, ITree
     return <div>{this._createTreeChart()}</div>;
   }
 
+  private _onWidthChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({ _layoutWidth: parseInt(e.target.value, 10) });
+  };
+
   private _createTreeChart(): JSX.Element {
     return (
       <>
+        <label htmlFor="changeWidth_Basic">Change Width:</label>
+        <input
+          type="range"
+          value={this.state?._layoutWidth}
+          min={65}
+          max={90}
```

Comment:
have you defined the min value inside tree chart code as well. If less than 65 causes the tree to distort, then define this minimum value in the main code.